test/functional/api/vim_spec.lua

```diff
@@ -487,6 +489,31 @@ describe('API', function()
     end)
   end)
 
+  describe('nvim_set_current_dir', function()
+    local start_dir
+
+    before_each(function()
+      clear()
+      lfs.mkdir("testdir")
```

Comment:
typical pattern for test files is to prefix with "X". 
```suggestion
      funcs.mkdir("Xtestdir")
```